CHANGES.md

```diff
@@ -3,6 +3,7 @@
 # Changes #
 
 * @dev (????-??-??)
+  * fixed off grid components bug
```

Comment:
Perhaps better wording?

    Fixed off grid components bug that could lead to OutOfMemory error.

Reply:
thanks!